eggplant/profiles/models.py

```diff
@@ -4,7 +4,16 @@
 from django.dispatch.dispatcher import receiver
 from django.db.models.signals import post_save
 from django.contrib.auth.models import User
+from django.core.files.storage import FileSystemStorage
+import os
 
+# Prepare profile photo dir
+profile_photo_dir = settings.BASE_DIR + "/media/profile/"
+
+if not os.path.exists(profile_photo_dir):
+    os.makedirs(profile_photo_dir)
+
+profile_photo_fs = FileSystemStorage(location=profile_photo_dir)
```

Comment:
I have taken this code out and it should use whatever the default is. Not sure if the folder needs to be made on the live server or anything like that :)